### PR TITLE
fixing the indexing

### PR DESCRIPTION
**Pull request by safwanrahman**
@ericholscher It should fix the search indexing!

### CODE REVIEW

File: readthedocs/search/management/commands/reindex_elasticsearch.py [hunk 1]
```diff
@@ -19,16 +19,16 @@ class Command(BaseCommand):
...
 
-        for chunk in chunked_queryset:
+        for chunk in chunks:
             data = {
                 'app_label': app_label,
                 'model_name': model_name,
                 'document_class': document_class,
                 'index_name': index_name,
-                'objects_id': list(chunk)
+                'objects_id': chunk
```
Comment (ericholscher) [inline readthedocs/search/management/commands/reindex_elasticsearch.py]: This should be named `chunk` I think?

**Comment by ericholscher** [comment]
I think we probably need to support both a list of object ID's for normal running, and also a start/end for reindexing perhaps

**Comment by safwanrahman** [comment]
@ericholscher Oops! I missed that!
updated to support both chunk and objects id.

File: readthedocs/search/tasks.py [hunk 2]
```diff
@@ -68,14 +68,26 @@ def switch_es_index(app_label, model_name, index_name, new_index_name):
 
 
 @app.task(queue='web')
-def index_objects_to_es(app_label, model_name, document_class, index_name, objects_id):
+def index_objects_to_es(app_label, model_name, document_class, index_name,
+                        chunk=None, objects_id=None):
+
+    assert not chunk and objects_id, "You can not pass both chunk and objects_id"
```
[the remark this answers is not in the document]
Comment (safwanrahman) [reply readthedocs/search/tasks.py]: @ericholscher fixed!

**Comment by ericholscher** [comment]
👍

**Pull request merged**